index.bs

```diff
@@ -338,13 +338,16 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       <dfn>serialized-policy-directive</dfn> = <a>feature-identifier</a> RWS <a>allow-list</a>
       <dfn>feature-identifier</dfn> = 1*( ALPHA / DIGIT / "-")
       <dfn>allow-list</dfn> = <a>allow-list-value</a> *(RWS <a>allow-list-value</a>)
-      <dfn>allow-list-value</dfn> = <a>serialized-origin</a> / "*" / "'self'" / "'src'" / "'none'"
+      <dfn>allow-list-value</dfn> = <a>serialized-origin</a> / <a>serialized-origin-with-wildcard-subdomain</a> / "*" / "'self'" / "'src'" / "'none'"
     </pre>
     <p><dfn><code>serialized-origin</code></dfn> is the
     <a>serialization of an origin</a>. However, the code points U+0027 ('),
     U+0021 (*), U+002C (,) and U+003B (;) MUST NOT appear in the serialization.
     If they are required, they must be percent-encoded as "`%27`", "`%2A`",
     "`%2C`" or "`%3B`", respectively.</p>
+    <p><dfn><code>serialized-origin-with-wildcard-subdomai</code></dfn> is a
+    <a>serialized-origin</a> with (1) a <a>registrable domain</a> for the <a>host</a>
```

Comment:
I think it's problematic to say that a `serialized-origin-with-wildcard` *is a* `serialized-origin`, when it explicitly breaks some of the conditions that a `serialized-origin` has (namely, that `*` cannot appear in it.)

Maybe we can say that it is the serialization of an origin whose host is a domain which is the concatenation of the string "*." and a domain which has a non-null registrable domain.

That's a little bit odd, since a domain isn't really supposed to have a '*' in it, but it's probably the best we can do without going the CSP route and defining a full syntax for it.

I don't think you want to require the remainder after the "*." to be a registrable domain on its own, since that would preclude things like `*.staging.mywebsite.example` -- in that case, `staging.mywebsite.example` isn't a registrable domain (`mywebsite.example` is).